x/ethbridge/types/genesis.go

```diff
@@ -0,0 +1,36 @@
+package types
```

Comment:
I would appreciate if we could remove any code from this PR that touches the genesis state. We should not be touching genesis state.

Reply:
The whole file should be removed, forget it. will remove it